e2e-polybft/e2e/consensus_test.go

```diff
@@ -101,21 +101,17 @@ func TestE2E_Consensus_Bulk_Drop(t *testing.T) {
 
 func TestE2E_Consensus_RegisterValidator(t *testing.T) {
 	const (
-		validatorSize  = 5
-		epochSize      = 5
-		epochReward    = 1000000000
-		premineBalance = "0x1A784379D99DB42000000" // 2M native tokens (so that we have enough balance to fund new validator)
+		validatorSize = 5
+		epochSize     = 5
+		epochReward   = 1000000000
 	)
 
 	var (
-		firstValidatorDataDir   = fmt.Sprintf("test-chain-%d", validatorSize+1) // directory where the first validator secrets will be stored
-		secondValidatorDataDir  = fmt.Sprintf("test-chain-%d", validatorSize+2) // directory where the second validator secrets will be stored
-		newValidatorInitBalance = "500000000000000000000000"                    // 500k - balance which will be transferred to the new validator
-		newValidatorStakeRaw    = "0x8AC7230489E80000"                          // 10 native tokens  - amount which will be staked by the new validator
-	)
+		firstValidatorDataDir  = fmt.Sprintf("test-chain-%d", validatorSize+1) // directory where the first validator secrets will be stored
+		secondValidatorDataDir = fmt.Sprintf("test-chain-%d", validatorSize+2) // directory where the second validator secrets will be stored
 
-	newValidatorStake, err := types.ParseUint256orHex(&newValidatorStakeRaw)
-	require.NoError(t, err)
+		premineBalance = ethgo.Ether(2e6) // 2M native tokens (so that we have enough balance to fund new validator)
```

Comment:
Constant for `2e6` ?

Reply:
I don't see any benefit in doing so. Not that code will be any more verbose than this. Only what I could have done is not use scientific notation if that's why you've suggested to extract a constant?